### PR TITLE
Clarify instructions for cloning to dom0

### PR DESCRIPTION
**Pull request by pierwill**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -134,15 +134,17 @@ That process is a little tricky, but here's one way to do it: assuming this code
...
+(Be sure to include the space after the parent directory.)
 After that initial manual step, the code in your development VM may be copied into place on `dom0` by setting the `SECUREDROP_DEV_VM` and `SECUREDROP_DEV_DIR` environmental variables to reflect the VM and directory to which you've cloned this repo, and running `make clone` from the root of the project on `dom0`:
 
 ```
-export SECUREDROP_DEV_VM=sd-dev    # set to your dev VM
-export SECUREDROP_DEV_DIR=/home/user/projects/securedrop-workstation    # set to your working directory
-make clone
+[dom0]$ export SECUREDROP_DEV_VM=sd-dev    # set to your dev VM
+[dom0]$ export SECUREDROP_DEV_DIR=/home/user/projects/securedrop-workstation    # set to your working directory
+[dom0]$ cd projects/securedrop-workstation
```
Comment (zenmonkeykstop) [inline README.md]: There's a note in the README that the dom0 home for the code must be /home/user/securedrop-workstation. Not sure if it's still a requirement, but this command would contradict that.

File: README.md [hunk 2]
```diff
@@ -134,15 +134,17 @@ That process is a little tricky, but here's one way to do it: assuming this code
 
     qvm-run --pass-io sd-dev 'tar -c -C /home/user/projects securedrop-workstation' | tar xvf -
 
+(Be sure to include the space after the parent directory.)
```
Comment (zenmonkeykstop) [inline README.md]: Might be worth clarifying what you mean by `parent directory`, or providing the example path explicitly - this is the first time it's mentioned.

**Comment by pierwill** [comment]
Force-pushed those changes!

**Comment by conorsch** [comment]
Thanks for submitting, @pierwill! To others on the team: is it worth preserving the "projects/" string in the example dir names? I'm not aware of anyone actually using that pardir—I suspect most folks are cloning to `sd-dev:/home/user/securedrop-workstation` and the same location in dom0. If so, let's use those simpler paths in the docs to make it as simple as possible to get started.

**Comment by zenmonkeykstop** [comment]
I think it's worth keeping (though I have checked into `/home/user/securedrop-workstation`), as it does allow for people following their own conventions/habits for organizing their work.

**Pull request merged**